unstract/connectors/src/unstract/connectors/filesystems/sftp/sftp.py

```diff
@@ -64,11 +67,27 @@ def can_read() -> bool:
     def get_fsspec_fs(self) -> SFTPFileSystem:
         return self.sftp_fs
 
+    @staticmethod
+    def get_json_schema() -> str:
+        f = open(f"{os.path.dirname(__file__)}/static/json_schema.json")
+        schema = f.read()
+        f.close()
+        return schema
+
+    @staticmethod
+    def requires_oauth() -> bool:
+        return False
+
+    @staticmethod
+    def python_social_auth_backend() -> str:
+        return ""
```

Comment:
@chandrasekharan-zipstack this I want to have a discussion with you.
If I don't inherit `requires_oauth` and `python_social_auth_backend`, then sftp doesn't load.
If I don't inherit `get_json_schema` , then json-schema of sftp doesn't load

Reply:
That's strange man, can you add a TODO comment to review if those 3 functions are needed. For some of the implementations we don't add it but it works. The whole point of inheritance is to avoid such code duplication